server/supernova/CMakeLists.txt

```diff
@@ -162,7 +162,7 @@ elseif(AUDIOAPI STREQUAL portaudio)
     target_include_directories(libsupernova PUBLIC ${PORTAUDIO_INCLUDE_DIRS})
     if(WIN32)
         target_compile_definitions(libsupernova PUBLIC PORTAUDIO_BACKEND -DPORTAUDIO_BACKEND)
-        target_link_libraries(libsupernova portaudio)
+        target_link_libraries(libsupernova portaudio_static)
```

Comment:
here also, you could eliminate this if() -- `${PORTAUDIO_DEFINITIONS}` never expands to anything, and `target_compile_definitions(libsupernova PUBLIC PORTAUDIO_BACKEND -DPORTAUDIO_BACKEND)` is exactly equivalent to `target_compile_definitions(libsupernova PUBLIC PORTAUDIO_BACKEND)`